src/fetchers.js

```diff
@@ -219,7 +219,10 @@ function fetchfallbacktasks(infos, wskOpts, contentPromise, staticPromise) {
  */
 function fetchactiontasks(infos, wskOpts, contentPromise, params) {
   return infos.map((info) => contentPromise.then((contentOpts) => {
-    const actionname = `${contentOpts.package || 'default'}/${info.selector ? `${info.selector}_` : ''}${info.ext}`;
+    const actionname = `${contentOpts.package
```

Comment:
what is the point of this? if the calculated action name doesn't match the requested selector or extension, we don't need to do the request at all and just give a 404.

Reply:
Quietly cleaning things up is easier than rejecting it, so I'm going with the easier option.

Reply:
why do we need to clean up at all ?

Reply:
So that the Epsagon drop-down is easier to use.

Reply:
*sigh*